src/main/java/com/autotune/common/k8sObjects/Metric.java

```diff
@@ -16,13 +16,17 @@
 
 import com.autotune.common.data.metrics.EMMetricResult;
 import com.autotune.common.performanceProfiles.AggregationFunctions;
```

Comment:
AggregationFunctions are only used here in Metric.java and should be in the same dir and not inside `performanceProfiles`

Reply:
Done